pkg/sif/update.go

```diff
@@ -117,7 +117,19 @@ func Update(fi *sif.FileImage, ii v1.ImageIndex, opts ...UpdateOpt) error {
 	}
 
 	// Write the new RootIndex into the SIF.
-	return f.writeBlobToFileImage(bytes.NewReader(ri), true)
+	return f.writeRootIndex(bytes.NewReader(ri))
+}
+
+// Update is a convenience function, for backward compatibility, which calls
+// OCIFileImage.UpdateRootIndex against the sif.FileImage fi.
+//
+// Deprecated: Use OCIFileImage.Update instead.
```

Comment:
Need to update the name of the replacement here. And I'd say let's put some strong language in about removing in the future, since this is a v0.x module still.

```suggestion
// Deprecated: Use OCIFileImage.UpdateRootIndex instead. Update will be removed in a future version.
```